src/components/shared/sidebar/VersionOverviewModal.tsx

```diff
@@ -13,11 +13,9 @@ import { MemoIconArrowRight } from "@/submodules/react-components/components/ker
 export default function VersionOverviewModal() {
     const versionOverviewData = useSelector(selectCachedValue(CacheEnum.VERSION_OVERVIEW));
 
-    const [preparedValues, setPreparedValues] = useState([]);
-
-    useEffect(() => {
-        if (!versionOverviewData) return;
-        setPreparedValues(prepareTableBodyVersionOverview(versionOverviewData));
+    const preparedValues = useMemo(() => {
+        if (!versionOverviewData) return [];
+        return prepareTableBodyVersionOverview(versionOverviewData);
```

Comment:
the function viewEvalSetRecordsModal already handeles null values so can be changed to something like

`useMemo(()=>prepareTableBodyVersionOverview(versionOverviewData),[versionOverviewData])`